src/pages/index copy.js

```diff
@@ -0,0 +1,76 @@
+import {nanoid} from 'nanoid';
```

Comment:
I'm not sure if `index copy` is a good name for this file. 😅  And this file contains content that you don't need. Therefore, do you even need this file?

Reply:
This was in our Project Template.
But indeed this has nothing to do on Github.
I will remove it 😅